mozfun/bitmask/bitmask_365/udf.sql

```diff
@@ -0,0 +1,6 @@
+/*
+A bitmask for 365 bits
+*/
+CREATE OR REPLACE FUNCTION bitmask.bitmask_365() AS (
+  CONCAT(b'\x1F', REPEAT(b'\xFF', 45))
+);
```

Comment:
Are the `bitmask` UDFs used outside of queries?

Reply:
I doubt it, but cc @fbertsch. I think we would need a tighter naming scheme for these before putting in mozfun.

Reply:
Agreed, I don't think anyone is using these at the moment. I can check in with the search team today though.